Tests/ProjectDescriptionTests/CompatibleXcodeVersionsTests.swift

```diff
@@ -20,4 +20,22 @@ final class CompatibleXcodeVersionsTests: XCTestCase {
         // Then
         XCTAssertCodable(subject)
     }
+
+    func test_codable_when_exact() {
+        // Given
+        let subject = CompatibleXcodeVersions.exact("13.2.1")
+
+        // Then
+        XCTAssertCodable(subject)
+    }
+
+    func test_codable_when_upToNext() {
+        // Given
+        let subject = CompatibleXcodeVersions.upToNextMajor("13.2")
+        let subject2 = CompatibleXcodeVersions.upToNextMinor("13.2")
+
+        // Then
+        XCTAssertCodable(subject)
+        XCTAssertCodable(subject2)
+    }
```

Comment:
Can you add also `isCompatible` tests, covering the various edge cases?

Reply:
I'm not sure what you're looking for here; `isCompatible` is being tested in the `EnvironmentLinterTests` which were added.

Also, now that `Codable` conformance is being synthesized, I don't think these tests serve any purpose. The `ProjectDescription` version of `CompatibleXcodeVersions` is just a struct with cases and initializers now. Would you object to just removing this file?

Reply:
Yeah I personally don't see the point in testing `Coddle` IMO. But I know the project has these tests, would be in favor of removing these going forward.